### PR TITLE
Feat(eos_cli_config_gen): Add support for qos map exp

### PR DESCRIPTION
**Pull request by philippebureau**
## Change Summary

add knob in eos_cli_config_gen to configure qos map exp 

## Related Issue(s)

N/A

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
adding a key under qos.map for exp field

Addition to qos.schema:
```
type: dict
  keys:
    qos:
      map:
          exp:
            type: list
            items:
              type: str
              description: |
                Example "0 to traffic-class 0"
```

## How to test

See molecule test 

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.jsonschema.json [hunk 1]
```diff
@@ -11335,6 +11335,14 @@
               },
               "title": "DSCP"
             },
+            "exp": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "description": "Example \"0 to traffic-class 0\"\n"
+              },
+              "title": "Exp"
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.jsonschema.json]: Please add exp -> EXP in `ansible-avd/ansible_collections/arista/avd/plugins/plugin_utils/schema/key_to_display_name.py`.

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/qos.md [hunk 2]
```diff
@@ -241,6 +241,7 @@ qos map dscp 46 to traffic-class 5
 qos map traffic-class 1 to dscp 56
 qos map traffic-class 2 4 5 to cos 7
 qos map traffic-class 6 to tx-queue 2
+qos map exp 0 to traffic-class 0
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/qos.md:245]: You should add documentation for this type of mapping similar to the others.
Hint: `ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/qos.j2`.

Comment (philippebureau) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/qos.md:245]: added with new commit

**Review by carl-baillargeon — APPROVED** [review]
LGTM. Tested locally

**Comment by ClausHolbechArista** [comment]
@philippebureau I have fixed a couple of issues. The things with blank lines are old issues, but since we are touching the template I wanted to get it fixed here. You also had a copy/paste error for the `if` condition which would have been hard to catch later.

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**